### ISSUE TITLE
keyboard navigation in menu gadget doesn't work properly

### ISSUE THREAD

**Issue by admich** [McCLIM/McCLIM#1262]
In menu gadget the keyboard navigation doesn't work properly. It seems that both key press and key release are handled with the result that the focus is moved with steps of two items.
